src/droplet.F

```diff
@@ -3265,10 +3268,12 @@ subroutine droplet_injection(pdata,xf,yf,dt,time_since_inject,mtime,my_reintro)
 
 #ifdef MPI
       !For deciding whether to inject more:
-      call mpi_allreduce(my_reintro+nparcelsLocalActive,tot_after_reintro,1,mpi_int,mpi_sum,mpi_comm_world,ierr)
+      call mpi_allreduce(real(my_reintro+nparcelsLocalActive), &
```

Comment:
Same comment as above.  Is there no way to get 8-byte integers working?

Reply:
Revised as suggested.